src/main/java/com/google/cloud/teleport/templates/BigQueryToBigtable.java

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Is there a reason for making this a Classic Template rather than a Flex Template?

Note that, from the root directory, Classic Templates are defined under src/, which Flex Templates are defined under v2/.